alembic/versions/c1913631e915_added_stats_to_anilist_table.py

```diff
@@ -0,0 +1,30 @@
+"""Added Stats to Anilist Table
```

Comment:
Can we merge these two migrations?

A quick and easy way to do it is to reset the db, load the migrations on master, checkout to this branch and delete these two migration files, then use the create migration command.